src/components/Calculator.js

```diff
@@ -0,0 +1,50 @@
+export default function Calculator() {
+  return (
+    <div className="wrapper">
+      <Screen />
+      <Buttons />
+    </div>
+  );
+}
+
+function Screen() {
+  return (
+    <div className="screen">
+      <p>0</p>
+    </div>
+  );
+}
+
```

Comment:
- [ OPTIONAL ] Great job working on the functionality so far but I think it would be a great idea if you just add each child in a separate component then you can import them to the calculator component since here we can achieve the important of reusable component in React 💪🏻💯✨⭐

Reply:
Hi @Reem-lab,

Thank you for your informative feedback. I will make changes 😊